digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadwayChangesDAO.scala

```diff
@@ -207,9 +207,9 @@ class RoadwayChangesDAO {
       // Checking sourceDiscontinuity
       (((previousRow.sourceDiscontinuity == nextRow.sourceDiscontinuity || previousRow.sourceDiscontinuity.isEmpty) && previousRow.sourceDiscontinuity.contains(Discontinuity.Continuous.value))
         || (previousRow.sourceDiscontinuity.contains(Discontinuity.Continuous.value) && !nextRow.sourceDiscontinuity.contains(Discontinuity.Continuous.value))) && nextRow.sourceDiscontinuity.get != Discontinuity.ParallelLink.value  &&
-      // Checking targetDiscontinuity
-      (((previousRow.targetDiscontinuity == nextRow.targetDiscontinuity || previousRow.targetDiscontinuity.isEmpty) && previousRow.targetDiscontinuity.contains(Discontinuity.Continuous.value))
-        || (previousRow.targetDiscontinuity.contains(Discontinuity.Continuous.value) && !nextRow.targetDiscontinuity.contains(Discontinuity.Continuous.value))) && nextRow.targetDiscontinuity.get != Discontinuity.ParallelLink.value
+        // Checking targetDiscontinuity
+        (((previousRow.targetDiscontinuity == nextRow.targetDiscontinuity || previousRow.targetDiscontinuity.isEmpty) && previousRow.targetDiscontinuity.contains(Discontinuity.Continuous.value))
+          || (previousRow.targetDiscontinuity.contains(Discontinuity.Continuous.value) && !nextRow.targetDiscontinuity.contains(Discontinuity.Continuous.value))) && nextRow.targetDiscontinuity.get != Discontinuity.ParallelLink.value
     }
```

Comment:
It is not clear to me what is the evaluation order for || and &&. More parenthesis might make it more clear.

Reply:
there are changes and fix for that piece of code in VIITE-2415 PR